api/lib/application/sessions/index.js

```diff
@@ -861,6 +861,36 @@ const register = async function (server) {
         ],
       },
     },
+    {
+      method: 'PATCH',
+      path: '/api/sessions/{id}/dismiss-live-alert',
+      config: {
+        validate: {
```

Comment:
L'API manque un peu de documentation

```suggestion
      config: {
        plugins: {
          'hapi-swagger': {
            produces: ['application/json'],
            consumes: ['application/json'],
          },
        },
        response: {
          failAction: 'log',
          status: {
            204: Joi.string().empty(''),
            401: responseObjectErrorDoc,
            403: responseObjectErrorDoc,
          },
        },
        validate: {
```

J'ai d'ailleurs dû fouiller dans le controller pour trouver que la réponse nominale était une 204.